doc/manual/manual.tex

```diff
@@ -8485,23 +8488,249 @@ \subsubsection{The van Keken thermochemical composition benchmark}
+algorithm in ASPECT.
+In fact, this problem is particularly well-suited to being computed with the VOF method, since 
+it consists of two distinct, immiscible fluids and interface tracking algorithms are 
+specifically designed not to allow the two fluids to mix.
+In particular, assuming the computation is sufficiently well-resolved, the fluids will not mix 
+at sub-grid scales over the entire duration of the computation.
+However note that this implies that all computations of the van Keken problem made with the VOF 
+method must necessarily be with discontinuous initial conditions.
+Finally, since one is often interested in a high resolution image of the shape of the interface 
+between the two fluids, one can use the output of the VOF method to examine the interface 
+within individual cells or within regions consisting of groups of cells.
+
+Another advantage the VOF method has over modeling moving interface problems with compositional 
+fields is that for problems in which the interface occupies a relatively small part of the 
+computational domain, all of the computational work in the VOF method is done in cells that lie 
+in a neighborhood of the interface, rather than in the entire computational domain. 
+
+\begin{figure}[b!]
+   \centering
+\includegraphics[width=0.5\textwidth]{cookbooks/benchmarks/vof-van-keken/rms_vel_comparison.png}
+   \caption{Evolution of the root mean square velocity as a function of time for computations 
+    of the Van Keken problem made with the VOF interface tracking algorithm with five 
+    different global mesh refinements.
+    Since the VOF initial conditions are discontinuous, the above results should really be 
+    compared with the computations with discontinuous initial conditions on the left in 
+    Figure~\ref{fig:vk-2}, although the above results also compare extremely favorably with the 
+    computations with smoothed, continuous initial conditions for the compositional field on 
+    the right in Figure~\ref{fig:vk-2}.
+    As in Figure~\ref{fig:vk-2}, 5 global refinements correspond to a $32 \times 32$ mesh and 10 global refinements correspond to a $1024 \times 1024$ mesh.}
+    \label{fig:vof-vk-1}
+\end{figure}
+
+\begin{figure}
+    \centering
+    \includegraphics[width=0.4\textwidth]{cookbooks/benchmarks/vof-van-keken/vof_van_keken_refinement_comparison.png}
+    \caption{\it The results of two computations of the Van Keken problem made with the VOF 
+        interface tracking algorithm  overlaid upon each other at $t_{end}=2000$.
+        This visualization consists of the \underline{two} zero contours of 
+        $\tilde{\psi}^7 \, (t_{end})$ and $\tilde{\psi}^8 \, (t_{end})$, where 
+        $\tilde{\psi}^7 \, (t_{end})$ is field where the zero contour is the reconstructed interface 
+        at the final time $t_{end}$ computed on a uniform grid with a refinement level of 7 and 
+        similarly for $\tilde{\psi}^8 \, (t_{end})$.
```

Comment:
This sentence was quite hard to follow. Could you break into two shorter sentences and define `\psi` towards the beginning?